bundles/org.openhab.binding.mikrotik/README.md

```diff
@@ -0,0 +1,371 @@
+* <kbd>Switch</kbd> `enabled` _(read-only)_ - Reflects enabled or disabled state
+* <kbd>Switch</kbd> `connected` _(read-only)_ - Reflects connected or disconnected state
+* <kbd>DateTime</kbd> `lastLinkDownTime` _(read-only)_ - Last time when link went down
+* <kbd>DateTime</kbd> `lastLinkUpTime` _(read-only)_ - Last time when link went up
+* <kbd>Number</kbd> `linkDowns` _(read-only)_ - Amount of link downs
+* <kbd>Number</kbd> `txRate` _(read-only)_ - Rate of data transmission in megabits per second
+* <kbd>Number</kbd> `rxRate` _(read-only)_ - Rate of data receiving in megabits per second
+* <kbd>Number</kbd> `txPacketRate` _(read-only)_ - Rate of data transmission in packets per second
+* <kbd>Number</kbd> `rxPacketRate` _(read-only)_ - Rate of data receiving in packets per second
+* <kbd>Number</kbd> `txBytes` _(read-only)_ - Amount of bytes transmitted
+* <kbd>Number</kbd> `rxBytes` _(read-only)_ - Amount of bytes received
+* <kbd>Number</kbd> `txPackets` _(read-only)_ - Amount of packets transmitted
+* <kbd>Number</kbd> `rxPackets` _(read-only)_ - Amount of packets received
+* <kbd>Number</kbd> `txDrops` _(read-only)_ - Amount of packets dropped during transmission
+* <kbd>Number</kbd> `rxDrops` _(read-only)_ - Amount of packets dropped during receiving
+* <kbd>Number</kbd> `txErrors` _(read-only)_ - Amount of errors during transmission
+* <kbd>Number</kbd> `rxErrors` _(read-only)_ - Amount of errors during receiving
+
+Populated only for `ether` interfaces:
+* <kbd>String</kbd> `defaultName` _(read-only)_ - Interface factory name
+* <kbd>String</kbd> `rate` _(read-only)_ - Ethernet link rate
+* <kbd>String</kbd> `autoNegotiation` _(read-only)_ - Ethernet auto-negotiation status
+
+Populated only for `cap` interfaces:
+* <kbd>Number</kbd> `registeredClients` _(read-only)_ - Amount of clients registered to WiFi interface
+* <kbd>Number</kbd> `authorizedClients` _(read-only)_ - Amount of clients authorized by WiFi interface
+* <kbd>String</kbd> `uptime` _(read-only)_ - Duration while thing is up
+* <kbd>DateTime</kbd> `upSince` _(read-only)_ - Time when thing got up
+
+## Text Configuration Example
+
+**Change config options accordingly.**
+
+_things/mikrotik.things_
+
+```
+Bridge mikrotik:routeros:rb1 "My RouterBoard" [ host="192.168.0.1", port=8728, login="openhab", password="thatsasecret", refresh=10 ] {
+	Thing interface eth1 "Eth1" [ name="ether1" ]
+	Thing interface eth2 "Eth2" [ name="ether2-wan1" ]
+	Thing interface cap1 "Cap1" [ name="cap5" ]
+	Thing interface ppp1 "PPPoE1" [ name="isp-pppoe" ]
+	Thing interface tun1 "L2TPSrv1" [ name="l2tp-parents" ]
+	Thing wifiRegistration wifi1 "Phone1" [ mac="F4:60:E2:C5:47:94", considerContinuous=60 ]
+	Thing wifiRegistration wifi2 "Tablet2" [ mac="18:1D:EA:A5:A2:9E" ]
+}
+```
+
+
+_items/mikrotik.items_
+
+```
+Group gRB1 "RB3011 System"
+Number   My_RB_3011_Free_Space     "Free space"     (gRB1) {channel="mikrotik:routeros:rb1:freeSpace"}
```

Comment:
I'm a little rusty on textual config but is it meant to be Number:DataAmount and not just Number now that the channels are using UOM?

Reply:
Right, I'll update readme as well.